src/open_inwoner/components/templates/components/Action/Actions.html

```diff
@@ -20,7 +20,7 @@
                 <div class="actions__actions">
                     {% button icon="file-pdf" text=_("Export to PDF") hide_text=True href="profile:action_export" uuid=action.uuid icon_outlined=True transparent=True %}
                     {% if action|is_connected:request.user %}
-                        {% dropdown icon="settings" %}
+                        {% dropdown icon="more_horiz" %}
                             <div class="dropdown__item">
```

Comment:
What is the rationale for replacing the wheel icon with dots? It has the same/similar functionality as the other settings (e.g. "Bewerken").

Reply:
@pi-sigma the reason: because it's in the UX design... and previously someone just picked the wrong icons. in Taiga you can see a link to the design files, but they re also here: https://taiga.maykinmedia.nl/project/open-inwoner/wiki/home - i will mark this comment as resolved.